### DISCUSSION TITLE
Nvidia Jetson (no not a FR)

### DISCUSSION THREAD

**Discussion by poweredgenl** [blakeblackshear/frigate#1154] [Ideas]
Out of curiousity - could this following setup work for me ? my goal would be to use a combi of frigate / deepstack / compreface on 1 system. 

My idea would be this: 

- Nvidia Jetson board / 4GB
- Google Coral USB accelerator
- Frigate in container/docker setup on Jetson with Coral (without the gpu acceleration - as this is not supported yet)
- Deepstack as container for GPU processing

Is this a realistic setup ? Esp is the performance 'okish' with just cpu of the jetson and the usb acceleator? Reason for this setup question -> i already have an ESX farm but cannot keep adding more usb devices etc...makes it complex...

**Comment by ozett** [comment]
jetson nano is done here. look issues or discussion for an example..

**Comment by cegit55** [comment]
Hi Peter,
Did you get round to doing this?
This is my dream setup ... but is implementing it a nightmare?
Nick
